modules/ROOT/pages/faq.adoc

```diff
@@ -310,3 +310,27 @@ xref:storage.adoc[Configuring Storage] page for examples.
 
 See the xref:faq.adoc#_why_is_the_dnsmasq_service_systemd_unit_masked[Why is the `dnsmasq.service` systemd unit masked]
 entry for an example config to unmask this unit.
+
+== How do I add the missing wifi-firmwares ?
+
+Some wifi-firmwares will be split into sub-packages in fedora 40.
+Fedora Cores OS will keep them in until fedora 41, but display a
+warning message in the console if NetworkManager-wifi is layered without
+any other wifi firmwares.
+
+If NetworkManager-wifi is a requested layered package, a warning is
+displayed to encourage users to also add a wifi-firmware.
```

Comment:
Here is a good place to mention the nuance of installing one versus all firwmares. any one system probably only needs one of the firmwares, but the tricky part is knowing which one.

So let's list some options here and then mention to them that if they don't know which one they need then just install them all. i.e.:

- `atheros-firmware` # Firmware for Qualcomm Atheros WiFi/Bluetooth adapters
- `realtek-firmware`: Firmware for Realtek WiFi/Bluetooth adapters

Reply:
we can list the ones we know about here in the instructions..

so we have the list.. then we say if you know the one you need then just:

```
rpm-ostree install --allow-inactive atheros-firmware
```

if you don't know the one you need then

```
rpm-ostree install --allow-inactive atheros-firmware realtek-firmware a b c d e f g
```

Reply:
ok, that's going to be a long command though.. I was hoping for a group-style thing I wasn't aware about :) 
will update

Reply:
yeah, if you want to break it up to make it look better you can like this:

```
rpm-ostree install --allow-inactive \
    atheros-firmware                \
    realtek-firmware                \
    ...
```